src/main/webapp/user_script.js

```diff
@@ -56,16 +54,17 @@ function loadFavItems() {
  * @param { Array } favoriteItems - the array of item ids
  */
 function populateFavoriteItemGrid(favoriteItems) {
-  const favItemContainer = $('#item-container');
+  const favItemContainer = $("#item-container");
+  const rowElem = $('<div class="row"></div>');
+  const MAX_CELLS_PER_ROW = 3;
   let currItemIndex = 0;
 
   while (currItemIndex < favoriteItems.length) {
- 
-    const rowElem = $('<div class="row"></div>');
-
-    const MAX_CELLS_PER_ROW = 3;
-
-    for (let cell = 0;cell < MAX_CELLS_PER_ROW && currItemIndex < favoriteItems.length; cell++, currItemIndex++) {
+    for (
+      let cell = 0;
+      cell < MAX_CELLS_PER_ROW && currItemIndex < favoriteItems.length;
+      cell++, currItemIndex++
+    ) {
       const item = favoriteItems[currItemIndex];
       const colElem = $('<div class="col-6 col-md-4"</div>');
```

Comment:
Is this missing a `>`?

```suggestion
      const colElem = $('<div class="col-6 col-md-4"></div>');
```